### PR TITLE
attendee/reg route update

### PR DESCRIPTION
**Pull request by jacobc2700**

### CODE REVIEW

File: src/services/events/events-router.ts
Comment (AydanPirani) [inline src/services/events/events-router.ts:28]: can this not all be cleaned up in one go: 

indOne({
    startTime: { $gte: now }
  }).sort({ startTime: 1 });

File: src/services/events/events-schema.ts
Comment (AydanPirani) [inline src/services/events/events-schema.ts:6]:
```suggestion
export const EventType = z.enum(["A", "B", "C"]);
```

File: src/services/events/events-schema.ts
Comment (AydanPirani) [inline src/services/events/events-schema.ts:63]: EventType.Values

**Review by AydanPirani — CHANGES_REQUESTED** [review]
a few changes, but lgtm otherwise!

**Review by AydanPirani — COMMENTED** [review]
lgtm! @divyack2  waiting on ur final approval

File: src/services/events/events-schema.ts
Comment (divyack2) [inline src/services/events/events-schema.ts:5]: wait should this be named with like CORPORATE, SPEAKER, etc instead of A, B, C?

**Review by AydanPirani — APPROVED** [review]

**Pull request merged**
